JavaScript-Runtime-(JSRT)-Reference.md

```diff
@@ -138,3 +139,20 @@ References of JSRT typedefs, constants, enumerations and APIs.
 * [[JsSetRuntimeMemoryLimit]]
 * [[JsStrictEquals]]
 * [[JsStringToPointer]]
+
+### New string API (Only non-Windows)
```

Comment:
I'm fine with accepting the PR btw. I can edit out the non-windows section and insert those APIs alphabetically in the reference table.